### PR TITLE
docs: update readme content

### PR DESCRIPTION
**Pull request by ryanseee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +26,47 @@ You can also opt to use npx:
...
+> In all the guides, we will refer to the CLI as `open-attestation` when running a command. That means we will assume the CLI is available in your execution path. If it's not the case, you will to change `open-attestation` by the full path to the executable.
+
+---
+
+## Usage
+
+#### List of features with the options available
+
+|                                            | Private Key | Wallet | Aws Kms |
+| ------------------------------------------ | ----------- | ------ | ------- |
```
Comment (simboonlong) [inline README.md:38]: the option columns are args that must be provided in order to use the feature yeah?

```suggestion
#### List of features with the required options

| Feature                                    | Private Key | Wallet | Aws Kms |
| ------------------------------------------ | ----------- | ------ | ------- |
```

File: README.md [hunk 2]
```diff
@@ -99,9 +149,13 @@ open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --
 open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --oav3
 ```
 
-## Document privacy filter
+:::note
+For transferable records, you should wrap them individually as each of them would be minted to a unique title escrow that represents the beneficiary and holder entities of the document. For more information about title escrow, refer [here](/docs/integrator-section/transferable-record/title-escrow).
+:::
```
Comment (simboonlong) [inline README.md:154]: :::note doesn't seem to render the markdown properly

File: README.md (hunk 1 above)
Comment (ryanseee) [reply README.md]: yup. ok have changed it.

File: README.md (hunk 2 above)
Comment (ryanseee) [reply README.md]: ya does not work in github readme. I have changed it to another one like a block of text. This way also works in the documentation-website.

File: README.md [hunk 3]
```diff
@@ -99,9 +149,11 @@ open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --
 open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --oav3
 ```
 
-## Document privacy filter
+> **_NOTE:_** For transferable records, you should wrap them individually as each of them would be minted to a unique title escrow that represents the beneficiary and holder entities of the document. For more information about title escrow, refer [here](/docs/integrator-section/transferable-record/title-escrow).
```
Comment (simboonlong) [inline README.md:152]: `(/docs/integrator-section/transferable-record/title-escrow)` dead link

Comment (ryanseee) [reply README.md]: ah ok i fixed it. the new link will also be broken, until I update the documentation-website to the new structure.

**Review by simboonlong — APPROVED** [review]

**Pull request merged**